src/rand.rs

```diff
@@ -25,11 +25,6 @@
 //! (seccomp filters on Linux in particular). See `SystemRandom`'s
 //! documentation for more details.
 
-#[cfg(feature = "nitro")]
-use nsm_api;
-#[cfg(feature = "nitro")]
-use nsm_lib;
```

Comment:
Why don't we need these anymore?

Reply:
I suspect https://github.com/veracruz-project/ring/commit/541f589ae3fab419ac9909d9b07858ef35d7fb54 might be part of the explanation, because we have a dependency on `nsm_api` and `nsm_lib` rather than the `aws-nitro-enclaves-nsm-api` workspace.